### PR TITLE
Change atom strings to pulsar

### PR DESCRIPTION
**Pull request by mdibella-dev**
Fix to https://github.com/pulsar-edit/pulsar/issues/347

### CODE REVIEW

File: lib/notification-issue.coffee [hunk 1]
```diff
@@ -98,23 +98,23 @@ class NotificationIssue
...
-          packageMessage = 'Atom Core'
+          packageMessage = 'Pulsar Core'
 
         @issueBody = """
           <!--
-          Have you read Atom's Code of Conduct? By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://github.com/atom/.github/blob/master/CODE_OF_CONDUCT.md
+          Have you read Pulsar's Code of Conduct? By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://pulsar-edit.dev/docs/resources/conduct/
 
-          Do you want to ask a question? Are you looking for support? The Atom message board is the best place for getting support: https://discuss.atom.io
+          Do you want to ask a question? Are you looking for support? The Pulsar message board is the best place for getting support: https://github.com/orgs/pulsar-edit/discussions
```
Comment (Daeraxa) [inline lib/notification-issue.coffee]:
```suggestion
          Do you want to ask a question? Are you looking for support? See our community areas to get in contact: https://pulsar-edit.dev/community.html
```

**Comment by Daeraxa** [comment]
There is actually a question here as to if we want to keep the automatic reporting thing or not or if we just direct to GH issues.

**Review by confused-Techie — APPROVED** [review]
This looks fantastic! And thanks to getting to this! We always appreciate the contribution.

So @Daeraxa while I do agree that we should remove the actually checking GitHub for existing issue thing, maybe we leave that to another PR to get this one through.

But I do absolutely agree on getting rid of that functionality, mostly due to generally being inaccurate, and when checking old Atom issues didn't do much to deter duplicate issues.

---

But as for this PR here, it's simple to the point and successfully updates everything that it looks like we need to. So I'd approve getting it merged!

**Comment by mdibella-dev** [comment]
Glad I could help. :)

**Comment by confused-Techie** [comment]
Alright, as it's been over 3 weeks with no other activity on this one to address the rest of the conversation I'll go ahead and merge this one. As it does address an issue on the main pulsar repo. Thanks again for contributing @mdibella-dev

**Pull request merged**